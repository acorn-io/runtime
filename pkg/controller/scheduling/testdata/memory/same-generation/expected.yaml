kind: AppInstance
apiVersion: internal.acorn.io/v1
metadata:
  name: app-name
  namespace: app-namespace
  uid: 1234567890abcdef
spec:
  image: test
  memory:
    oneimage: 1048576 # 1Mi
status:
  defaults:
    memory:
      "": 0
      left: 0
      oneimage: 0
  scheduling:
    oneimage:
      requirements:
        limits:
          memory: 10Mi
        requests:
          memory: 10Mi
    left:
      requirements: {}
  namespace: app-created-namespace
  appImage:
    id: test
  appSpec:
    containers:
      oneimage:
        sidecars:
          left:
            image: "foo"
            ports:
              - port: 90
                targetPort: 91
                protocol: tcp
        ports:
          - port: 80
            targetPort: 81
            protocol: http
        image: "image-name"
        build:
          dockerfile: "Dockerfile"
          context: "."
  conditions:
    - type: scheduling
      reason: Success
      status: "True"
      success: true     
