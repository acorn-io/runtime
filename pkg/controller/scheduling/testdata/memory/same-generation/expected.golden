`apiVersion: internal.acorn.io/v1
kind: AppInstance
metadata:
  creationTimestamp: null
  name: app-name
  namespace: app-namespace
  uid: 1234567890abcdef
spec:
  image: test
  memory:
    oneimage: 1048576
status:
  appImage:
    buildContext: {}
    id: test
    imageData: {}
    vcs: {}
  appSpec:
    containers:
      oneimage:
        build:
          context: .
          dockerfile: Dockerfile
        image: image-name
        metrics: {}
        ports:
        - port: 80
          protocol: http
          targetPort: 81
        probes: null
        sidecars:
          left:
            image: foo
            metrics: {}
            ports:
            - port: 90
              protocol: tcp
              targetPort: 91
            probes: null
  appStatus: {}
  columns: {}
  conditions:
    reason: Success
    status: "True"
    success: true
    type: scheduling
  defaults:
    memory:
      "": 0
      left: 0
      oneimage: 0
  namespace: app-created-namespace
  scheduling:
    left:
      requirements: {}
    oneimage:
      requirements:
        limits:
          memory: 10Mi
        requests:
          memory: 10Mi
  staged:
    appImage:
      buildContext: {}
      imageData: {}
      vcs: {}
  summary: {}
`
